internal/integration/spec.go

```diff
@@ -91,7 +91,14 @@ func LoadSpecFiles(filesPath string) (Specs, error) {
 			continue
 		}
 		logrus.Debugf("spec file loaded for service:%s", sd.Service)
-		specs.SpecsByName[sd.Service] = sd
+		if spec, ok := specs.SpecsByName[sd.Service]; ok {
+			spec.Entities = append(spec.Entities, sd.Entities...)
+			specs.SpecsByName[sd.Service] = spec
```

Comment:
just curiosity, do you have to re-assign the variable in the map? Wouldn't it be sufficient to just append the entities?

Reply:
It was complaining that it was possible , but I will dig into that

Reply:
It's because the map references to a value and not a pointer. And there's no guarantee on the location of that value. I'll change that to a pointer

https://stackoverflow.com/questions/32751537/why-do-i-get-a-cannot-assign-error-when-setting-value-to-a-struct-as-a-value-i